### PR TITLE
Initiate mocha tests

### PR DESCRIPTION
**Pull request by leonfu103**

### CODE REVIEW

**Comment by slwen** [comment]
Is it worth deleting the Jest versions of the Mocha tests you've added in this PR?

File: package.json [hunk 1]
```diff
@@ -31,7 +31,9 @@
     "babel-preset-es2015": "^6.3.13",
     "babel-preset-react": "^6.3.13",
     "browserify": "~13.0.0",
+    "chai": "^3.5.0",
     "compression": "~1.6.1",
+    "enzyme": "^2.4.1",
```
Comment (slwen) [inline package.json]: I noticed this was installed but not used yet? Planning to use this shortly?

File: src/api/__specs__/campaigns-spec.js [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+      });
+
+      it('does not fetch results', () => {
+        assert.notCalled(spy);
+      });
+
+      it.skip('defers callback with empty results', () => {
+        assert.notCalled(callback);
+
+        // jest.runOnlyPendingTimers();
```
Comment (slwen) [inline src/api/__specs__/campaigns-spec.js]: small comment here

File: package.json (hunk 1 above)
Comment (leonfu103) [inline package.json]: Yes, will use it maybe in the next PR.

**Pull request merged**